conjureup/app.py

```diff
@@ -16,6 +16,7 @@
 
 import raven
 import yaml
+from juju.model import CharmStore
```

Comment:
Really need to refactor this in python-libjuju.  `juju.model` is the wrong place for it.  Ah well.